md/minimizer.py

```diff
@@ -160,3 +176,90 @@ def minimize_host_4d(mols, host_system, host_coords, ff, box) -> np.ndarray:
         assert np.all(norm < 25000)
 
     return final_coords[:num_host_atoms]
+
+
+def equilibrate_complex(
```

Comment:
As a note we should probably look at the equilibrate function I added recently and see if we can't merge this functionality

Reply:
the pre-equilibration protocol is slightly different for RABFE due to the blocker alignment, but yes we should still try and merge these two bits of code.